R/utils-misc.R

```diff
@@ -44,8 +44,27 @@ gts_mapper <- function(x, context) {
   purrr::as_mapper(x)
 }
 
-# test if the element is convertible to a function
-.is_convertible_as_function <- function(f) {
-  tryCatch(rlang::as_function(f), error = function(e) NULL) %>%
-    rlang::is_function()
-}
```

Comment:
FYI @larmarange 

I had to omit this function because I think there is a potential for a user issue. I ran into a similar issue when users would pass string function names in `add_p()`. In rare cases, the user will pass the function name as a string, but the string was created in an environment where that function existed. When we try to evaluate that string as a function name, we must evaluate it in the same environment the formula was created to avoid an error.